frontend/src/stores/search.ts

```diff
@@ -95,29 +107,33 @@ export function computeQueryParams(
     // e.g., { licenseTypes: [{ code: "commercial", checked: true }] }
     // => { license_type: "commercial" }
     ...filtersToQueryData(filters, searchType),
-  }
-
-  // `INCLUDE_SENSITIVE_QUERY_PARAM` is used in the API params, but not shown on the frontend.
-  const ffStore = useFeatureFlagStore()
-  if (mode === "API" && ffStore.isOn("fetch_sensitive")) {
-    searchQuery[INCLUDE_SENSITIVE_QUERY_PARAM] = "true"
+    ...getSensitiveQuery(mode),
   }
 
   return searchQuery
 }
 
-export function collectionToPath(collectionParams: CollectionParams) {
-  switch (collectionParams.collection) {
-    case "tag": {
-      return `tag/${collectionParams.tag}/`
-    }
-    case "creator": {
-      return `source/${collectionParams.source}/creator/${collectionParams.creator}/`
-    }
-    case "source": {
-      return `source/${collectionParams.source}/`
-    }
+// TODO: After the API changes are done, replace
+// `tags` with `unstable__tag`
+export function buildCollectionQuery(
+  collectionParams: CollectionParams,
+  mode: "frontend" | "API"
```

Comment:
Could you document this `mode` a bit to clarify specifically what it refers to? This is basically a helper that sometimes processes API urls and sometimes frontend urls? If the frontend version simply pulls out the params and returns them, maybe it doesn't need to use this helper at all. See [this example](https://github.com/WordPress/openverse/pull/3866/files#r1513515761).

Reply:
Removed in [5cf32fc](https://github.com/WordPress/openverse/pull/3866/commits/5cf32fc5f72742c60081d32a98938a15f1ed5f64)